test/utils/image/manifest.go

```diff
@@ -30,6 +30,7 @@ type RegistryList struct {
 	GcAuthenticatedRegistry string `yaml:"gcAuthenticatedRegistry"`
 	DockerLibraryRegistry   string `yaml:"dockerLibraryRegistry"`
 	DockerGluster           string `yaml:"dockerGluster"`
+        DockerTestSnapshot      string `yaml:"dockerTestSnapshot"`
```

Comment:
I think there's a change needed to add the registry to a case statement:
https://github.com/kubernetes/kubernetes/pull/91257/files#diff-fa4ddc33013d1e57259726a4c1c327903e058453aece262eecfb48300e946c93R292

Reply:
Done